.rubocop.yml

```diff
@@ -0,0 +1,2 @@
+Metrics/MethodLength:
+  Max: 30
```

Comment:
Was rubocop useful to run? I've never run  it on this code.

Reply:
Emacs actually runs it as you're editing, so it was helpful for me while I was writing the tests. It seems to be... very opinionated. This rule in particular was generating a lot of noise when looking through other RDL code, so I upped the line count.